spec/controllers/deploys_controller_spec.rb

```diff
@@ -66,6 +66,10 @@
         receive(:stderr)
           .and_return(StringIO.new('Something went wrong'))
       )
+      allow(ruby_terraform.configuration).to(
+        receive(:stdout)
+          .and_return(StringIO.new('Something happened\n'))
```

Comment:
nit: I'm thinking maybe we should improve this msg `Something happened` with some more precise?

Reply:
Well, this is just a UT, and follows the same `something` syntax used before.